internal/artifacts/mtad.go

```diff
@@ -67,6 +68,15 @@ func genMtad(mtaStr *mta.MTA, ep dir.ITargetArtifacts, deploymentDesc bool, plat
 		}
 	}
 
+	schemaVersionSlice := strings.Split(*mtaStr.SchemaVersion, ".")
```

Comment:
please let's define a function for this like fixSchemaVersion or adjust...

Reply:
Done